editor/components/block-settings-menu/reusable-block-delete-button.js

```diff
@@ -9,54 +9,54 @@ import { noop } from 'lodash';
 import { compose } from '@wordpress/compose';
 import { IconButton } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
-import { isSharedBlock } from '@wordpress/blocks';
+import { isReusableBlock } from '@wordpress/blocks';
 import { withSelect, withDispatch } from '@wordpress/data';
 
-export function SharedBlockDeleteButton( { sharedBlock, onDelete, itemsRole } ) {
-	if ( ! sharedBlock ) {
+export function ReusableBlockDeleteButton( { reusableBlock, onDelete, itemsRole } ) {
+	if ( ! reusableBlock ) {
 		return null;
 	}
 
 	return (
 		<IconButton
 			className="editor-block-settings-menu__control"
 			icon="no"
-			disabled={ sharedBlock.isTemporary }
-			onClick={ () => onDelete( sharedBlock.id ) }
+			disabled={ reusableBlock.isTemporary }
+			onClick={ () => onDelete( reusableBlock.id ) }
 			role={ itemsRole }
 		>
-			{ __( 'Delete Shared Block' ) }
+			{ __( 'Remove from Reusable Blocks' ) }
```

Comment:
This one is a bit confusing to me. I understand better "Remove reusable block" but I can see how the ambiguity there too.